tests/sentry/backup/snapshots/ReleaseTests/test_at_head.pysnap

```diff
@@ -1035,60 +1034,60 @@ source: tests/sentry/backup/test_releases.py
   pk: 1
 - fields:
     custom_dynamic_sampling_rule: 1
-    project: 4553577575940097
+    project: 4553773213548546
   model: sentry.customdynamicsamplingruleproject
   pk: 1
 - fields:
     application: 1
-    date_added: '2024-02-29T10:09:36.207Z'
-    expires_at: '2024-02-29T18:09:36.207Z'
-    hashed_refresh_token: null
-    hashed_token: null
+    date_added: '2024-04-03T23:22:48.648Z'
+    expires_at: '2024-04-04T07:22:48.648Z'
+    hashed_refresh_token: a583193f2a86dc46623cad1a66d9814073a203f94900aeecfa29c4dc8a841a90
+    hashed_token: 73fda1b8ed737e3e8f05d10056c2baeea910bcd55badd64b7a5ff2df78662e64
     name: null
-    refresh_token: 971306105618847f6b4a39713fc8d37a69e15ebca853d4270b0beb68e2b667c9
+    refresh_token: fc6f4c19c46a65da8350c0e3ac547147dd41111bf404c77fb2fcde3cfd2de4b7
     scope_list: '[]'
     scopes: '0'
-    token: 6e004cca469b388eab5476c3ab0f8c4ee641a087941ba96b362138d27470057f
-    token_last_characters: 057f
+    token: 35d6b2efb0c939b060042dd6de7d6190f9a0ad09d28d6f0b86bdb908867cfbd2
+    token_last_characters: fbd2
     token_type: null
     user: 5
   model: sentry.apitoken
   pk: 1
 - fields:
     application: 1
-    date_added: '2024-02-29T10:09:36.302Z'
+    date_added: '2024-04-03T23:22:48.694Z'
     expires_at: null
-    hashed_refresh_token: null
-    hashed_token: null
+    hashed_refresh_token: efc59fb159c93a92cfbcd8e8992b6260cc49514c6a7e90520fe48bba6cc10094
+    hashed_token: 8d3df431aefcf9be592c4c58e52892c8a53c880110a11409cf80ae509f7c5010
     name: create_exhaustive_sentry_app
-    refresh_token: 0c80e6dfad48434df6c2e82a1deb421daa7bce4142ef0198404099b8b88d08f7
+    refresh_token: eaa67897315f01aa9c69080e6d299a81012a485a2d2fe869875399e4edef37a8
     scope_list: '[]'
     scopes: '0'
-    token: 3cd22573eced4b8289beb5a68fde923231069e085a713fffa0b25273b224c9f4
-    token_last_characters: c9f4
+    token: c9954191fded44bc82f6e924c4224975a833f542e5aaa301ccdd31d588094c69
+    token_last_characters: 4c69
     token_type: null
```

Comment:
Intentional to have some tokens with no type, right?

Reply:
Yes, that's correct. To start we will only be populating the token_type for User Auth Tokens.